File: pytools/data/data.code.csharp.client/AckRoleLoginOk.cs
using System.Collections;
using System.Collections.Generic;


public class AckRoleLoginOk
{
	private string _uname;


	public AckRoleLoginOk(Packet packet)
	{
		this._uname = packet.ReadString();
	}


	public string uname
	{
		get { return this._uname; }
		set { this._uname = value; }
	}

}

File: pytools/data/data.code.golang.client/ack_role_login_ok.go
package proto

import (
	"packet"
)

type AckRoleLoginOk struct {
	uname                    string
}

func AckRoleLoginOkDecode(pack *packet.Packet) *AckRoleLoginOk {
	ackRoleLoginOk := &AckRoleLoginOk{}

	ackRoleLoginOk.uname = pack.ReadString()
	return ackRoleLoginOk
}

func (this *AckRoleLoginOk) GetUname() string {
	return this.uname
}
